packages/perseus/src/widgets/__shared__/noop-validator.tsx

```diff
@@ -0,0 +1,20 @@
+import type {PerseusScore} from "../../types";
+
+/**
+ * Several widgets don't have "right"/"wrong" validation logic,
+ * so this just says to move on past those widgets
+ *
+ * TODO: widgets that use this probably shouldn't have any
+ * validation logic and the thing validating an exercise
+ * should just know to skip these
```

Comment:
+++  I think there is already some logic in `renderer.tsx` that knows how to skip scoring a widget if it doesn't have `simpleValidate`. Maybe we should make this more explicit by adding a flag on `WidgetExports<T>` that declares if the widget is scorable.

https://github.com/Khan/perseus/blob/f083200340545f41275f0696dbfc967f45028b0c/packages/perseus/src/renderer.tsx#L1830